ios/magnet/AppDelegate.m

```diff
@@ -52,8 +54,17 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
   UIViewController *rootViewController = [UIViewController new];
   rootViewController.view = rootView;
   self.window.rootViewController = rootViewController;
+  [NotificationsHelper register];
   [self.window makeKeyAndVisible];
   return YES;
 }
 
+- (void)applicationDidBecomeActive:(UIApplication *)application {
+  [NotificationsHelper clearNotifications];
+}
+
+- (void)applicationDidEnterBackground:(UIApplication *)application {
+  [NotificationsHelper clearNotifications];
```

Comment:
I don't mind really. I think eventually, once we know how all the parts work, we should create a ReactNative JS abstraction module around both android and iOS and control notifications from there.

```js
Notifications.enable();
Notifications.disable();
Notifications.clear();
```